src/api/sip.ts

```diff
@@ -191,24 +221,24 @@ export class SIP extends EventEmitter {
       '/JsSIP ' +
       jssipVersion
     //
-    const callOptions = ((sipLoginOption.pbxTurnEnabled && turnConfig) ||
-      {}) as CallOptions
-    if (!callOptions.pcConfig) {
-      callOptions.pcConfig = {}
+    const callOptions = ((pbxTurnEnabled && turnConfig) || {}) as CallOptions
+    let { pcConfig } = callOptions
+    if (!pcConfig) {
+      pcConfig = {}
     }
-    if (!Array.isArray(callOptions.pcConfig.iceServers)) {
-      callOptions.pcConfig.iceServers = []
+    if (!Array.isArray(pcConfig.iceServers)) {
+      pcConfig.iceServers = []
     }
     if (sipLoginOption.turnConfig) {
-      callOptions.pcConfig.iceServers.push(sipLoginOption.turnConfig)
+      pcConfig.iceServers.push(sipLoginOption.turnConfig)
```

Comment:
you manipulating the original object, instead you should create a copy of the object and update it.